code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -179,10 +179,9 @@
 	activated = TRUE
 	addtimer(CALLBACK(src, PROC_REF(give_larva)), XENO_PYLON_ACTIVATION_COOLDOWN, TIMER_UNIQUE|TIMER_OVERRIDE|TIMER_LOOP|TIMER_DELETE_ME)
 
+/// Looped proc via timer to give larva after time
 #define ENDGAME_LARVA_CAP_MULTIPLIER 0.4
 #define LARVA_ADDITION_MULTIPLIER 0.10
-
-/// Looped proc via timer to give larva after time
```

Comment:
Revert this incorrect autodoc change moving it somereason to ENDGAME_LARVA_CAP_MULTIPLIER.

Reply:
oops it was an accident